src/Behavior/DatasetAttrValue.php

```diff
@@ -21,13 +19,21 @@ class DatasetAttrValue implements AttrValueInterface
      */
     protected $dataset;
 
-    public function __construct(string ...$dataset)
+    /**
+     * @param string ...$dataset
+     */
+    public function __construct(...$dataset)
```

Comment:
not sure, whether that works already in PHP 5.5

Reply:
Good catch, that's PHP 5.6 and newer

Reply:
nope, only since 5.6, will update